internal/services/springcloud/spring_cloud_gateway_sso_resource.go

```diff
@@ -0,0 +1,242 @@
+package springcloud
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/springcloud/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/springcloud/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"github.com/tombuildsstuff/kermit/sdk/appplatform/2022-11-01-preview/appplatform"
```

Comment:
support for `appplatform` is available in `go-azure-sdk` - why is this using the legacy SDK? https://github.com/hashicorp/go-azure-sdk/tree/main/resource-manager/appplatform

Reply:
I was trying to migrate to `go-azure-sdk`, but blocked by https://github.com/hashicorp/go-azure-sdk/issues/200